polaris.shopify.com/package.json

```diff
@@ -10,8 +10,8 @@
   },
   "dependencies": {
     "@shopify/polaris": "^9.0.0",
+    "@shopify/polaris-tokens": "5.0.0-rc.1",
```

Comment:
Friendly reminder to release this RC before merging this PR. (I don't see it on [unpkg](https://unpkg.com/browse/@shopify/polaris-tokens@5.0.0-rc.1)